src/_pytest/doctest.py

```diff
@@ -108,13 +109,16 @@ def pytest_unconfigure():
     RUNNER_CLASS = None
 
 
-def pytest_collect_file(path, parent):
+def pytest_collect_file(
+    path: py.path.local, parent
```

Comment:
Type `parent`?
You've left it out below also, so there's likely a reason?

Reply:
I just haven't looked into what it's supposed to be. Sometimes to not get stuck too much and make progress, I leave some unrelated parameters blank, to hopefully be filled in the future.

BTW, I think one of the good things to do is go over all of the hookspecs and type annotate them, and then backfill all of the implementations with that. But the types are not always clear...